pkg/skuba/actions/node/remove/remove_test.go

```diff
@@ -18,42 +18,132 @@
 package remove
 
 import (
+	"crypto/sha1"
+	"fmt"
 	"testing"
 
-	v1 "k8s.io/api/core/v1"
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes/fake"
 )
 
 func Test_RemoveMasterNode(t *testing.T) {
```

Comment:
The test case seems not just for testing master node removing, also removing worker nodes included. It's better to use something like Test_RemoveNode.

Reply:
https://github.com/SUSE/skuba/pull/674